about.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="top-page">
+        <a href=""><img src="images/facebook-icon.svg" alt="facebook"></a>
+        <a href=""><img src="images/twitter-icon.svg"></a>
+        <a href="">Logout</a>
+        <a href="">Mypage</a>
+        <a href="">Korean</a>
+    </div>
+    <header>
+        <div class="hamburger">
+            <img src="images/hamburger-icon.svg" alt="menu">
+        </div>
+        <div class="top-right">
+            <div class="mo">
+                <a href="index.html"><img src="images/main-logo.svg" alt="logo"></a>
+            </div>
+        </div>
+        <nav class="nav">
+            <a href="about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">News</a>
+            <div class="campaign">
+                <a href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] [Optional but highly recommended] For better user experience, I would kindly advise you implement this change. I notice once i navigate to the about page of your project in the mobile view, the hamburger menu is not functioning which makes it difficult for me to navigate back to the home page. currently the hamburger menu is functioning for only the home page and once in the about page it's not functioning

> N.B: I marked this as `optional` because the issue was not raised by the previous reviewer but i would highly recommend that you should work on it before submitting for the next review. Thanks

About page mobile 👇 

![cor-abt](https://user-images.githubusercontent.com/78634841/225775322-be07b597-fe81-4d20-8136-ec2bbcc47818.png)